index.html

```diff
@@ -410,6 +413,9 @@ <h3>SigLens v/s ClickHouse</h3>
         <div class="code position-relative ingest-Cmd" id="generate-data">
           <span class="cmd-box" id="cmd-box">curl -Ls https://siglens.com/sendevents.sh | sh</span>
           <i class="fa-regular fa-clone position-absolute copy-btn" id="copyCmdBtn" data-toggle="tooltip"></i>
+          <svg class="check-mark-data" id="check-mark-data" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+            <polyline points="4 12 9 17 20 6" />
```

Comment:
same here, we should use the svg file from assets

Reply:
save as above